sito/static/html/classe.html

```diff
@@ -2,202 +2,210 @@
+                <thead>
+                    <tr>
+                        <th style="cursor: default;">Posizione</th>
+                        <th style="text-align:center;cursor: default">Squadra</th>
+                        <th style="text-align:center;z-index:1;cursor: default;color: #000000">Punti Totali</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for squadra,punti in classifica_squadre(classe,stagione_corrente).items()%}
+                    <tr style="background: rgb{{calcola_valore_rgb(squadra)}};">
+
+
+                        {% if loop.index==1 %}
+                        <td>
+                            <img class="img_top" src="{{ url_for('static',filename='images/primo.png')}}">
+                        </td>
+                        {% elif loop.index==2%}
+                        <td>
+                            <img class="img_top" src="{{ url_for('static',filename='images/secondo.png')}}">
+                        </td>
+                        {% elif loop.index==3%}
+                        <td>
+                            <img class="img_top" src="{{ url_for('static',filename='images/terzo.png')}}">
+                        </td>
+                        {% else %}
+                        <td>
+                            &#160&#160 {{loop.index}} 
+                        </td>
+                        {%endif%}
+                        <td style="text-align:center">
+                            {{ squadra}} 
+                            <img src="{{ url_for('static',filename='images/loghi/'+loghi[squadra])}}">
+                        </td>
+                        <td style="text-align:center">
+                            {{ punti}}
+                        </td>
+                    </tr>
+
+                    {% endfor %}
+                    
+                </tbody>
+            </table>
+        </section>
 
+        <section style="margin-top: 50px; width: 50%; float: left; margin-left: 40px;">
+            <canvas id="myChart" width="900" height="400"></canvas>
+        </section>
+    
     </main>
-  
-<script>
-    // Declare arrays for labels and data
-    var labels = [
-        {% for squadra,punti in classifica_squadre(classe, stagione_corrente).items() %}
-            "{{ squadra }}"{% if not loop.last %},{% endif %}
+
+    <script>
+        // Declare arrays for labels and data
+        var labels = [
+            {% for squadra, punti in classifica_squadre(classe, stagione_corrente).items() %}
+        "{{ squadra }}"{% if not loop.last %}, {% endif %}
         {% endfor %}
     ];
 
-    var data = [
-        {% for squadra,punti in classifica_squadre(classe, stagione_corrente).items() %}
-            {{ punti }}{% if not loop.last %},{% endif %}
+        var data = [
+            {% for squadra, punti in classifica_squadre(classe, stagione_corrente).items() %}
+        { { punti } } {% if not loop.last %}, {% endif %}
```

Comment:
```suggestion
        {{ punti }} {% if not loop.last %}, {% endif %}
```
questo dovrebbe sistemare la visione del grafico